src/test/system.rs

```diff
@@ -179,6 +183,16 @@ fn run_sccache_command_test(sccache: &Path, compiler: Compiler, tempdir: &Path)
         let expected_lines: Vec<_> = expected.lines().collect();
         assert_eq!(lines, expected_lines);
     }
+    if name == "gcc" {
+        trace!("test -MP with -Werror");
```

Comment:
I mean, the whole point here is simply that we fail the first compilation, so we retry, and then we muck up the commandline for the second compilation. It's sort of hard to make a non-contrived test case here.